packages/fuzzer/fuzzing_async.cpp

```diff
@@ -107,6 +117,10 @@ void CallJsFuzzCallback(Napi::Env env, Napi::Function jsFuzzCallback,
   // thread and continue with the next invocation.
 
   try {
+    if (setjmp(errorBuffer) != 0) {
+      std::cerr << SEGFAULT_ERROR_MESSAGE << std::endl;
+      exit(139);
```

Comment:
Will this exit cause libFuzzer report a finding? Extract `139` in a more readable constant?